config/Wasteland/CityLoot.cfg

```diff
@@ -80,8 +80,6 @@
         minecraft:bucket,1,1
         minecraft:bread,4,2
         minecraft:iron_ingot,2,1
-        minecraft:compass,1,1
-        minecraft:map,1,1
```

Comment:
why do you remove this?

Reply:
These items are pretty useless with Journeymap in game.

Reply:
well, i guess.. you are right.. well gonna accept this